keras_cv/models/backbones/efficientnet_v2/efficientnet_v2_backbone.py

```diff
@@ -0,0 +1,610 @@
+    return int(math.ceil(depth_coefficient * repeats))
+
+
+def get_block_conv(args, activation, survival_probability, name):
+    # Determine which conv type to use:
+    if args["conv_type"] == "mb_conv":
+        return MBConvBlock(
+            input_filters=args["input_filters"],
+            output_filters=args["output_filters"],
+            expand_ratio=args["expand_ratio"],
+            kernel_size=args["kernel_size"],
+            strides=args["strides"],
+            se_ratio=args["se_ratio"],
+            activation=activation,
+            bn_momentum=0.9,
+            survival_probability=survival_probability,
+            name=name,
+        )
+    elif args["conv_type"] == "fused_mb_conv":
+        return FusedMBConvBlock(
+            input_filters=args["input_filters"],
+            output_filters=args["output_filters"],
+            expand_ratio=args["expand_ratio"],
+            kernel_size=args["kernel_size"],
+            strides=args["strides"],
+            se_ratio=args["se_ratio"],
+            activation=activation,
+            bn_momentum=0.9,
+            survival_probability=survival_probability,
+            name=name,
+        )
+    raise ValueError(
+        "Expected `block_args['conv_type']` to be "
+        "one of 'mb_conv', 'fused_mb_conv', but got "
+        f"`block_args['conv_type']={args['conv_type']}`"
+    )
+
+
+@keras.utils.register_keras_serializable(package="keras_cv.models")
+class EfficientNetV2Backbone(Backbone):
+    """Instantiates the EfficientNetV2 architecture.
+
+    Args:
+        include_rescaling: bool, whether to rescale the inputs. If set
+            to `True`, inputs will be passed through a `Rescaling(1/255.0)`
+            layer.
+        width_coefficient: float, scaling coefficient for network width.
+        depth_coefficient: float, scaling coefficient for network depth.
+        default_size: integer, default input image size.
+        dropout_rate: float, dropout rate before final classifier layer.
+        drop_connect_rate: float, dropout rate at skip connections.
+        depth_divisor: integer, a unit of network width.
+        min_depth: integer, minimum number of filters.
+        activation: activation function.
+        kernel_size: List of integers, sizes to use for convolutional kernels in
+            each block.
+        num_repeats
+        block_args: list of dicts, parameters to construct block modules.
+        model_name: string, model name.
+        input_shape: optional shape tuple, defaults to (None, None, 3).
+        input_tensor: optional Keras tensor (i.e. output of `layers.Input()`)
+            to use as image input for the model.
+
+    Usage:
+    ```python
+    efficientnet = keras_cv.models.EfficientNetV2Backbone.from_preset(
+        "efficientnetv2-s"
+    )
+    images = tf.ones((1, 256, 256, 3))
+    outputs = efficientnet.predict(images)
+    ```
+    """
+
+    def __init__(
+        self,
+        *,
+        include_rescaling,
```

Comment:
Can we rename all list parameters `stackwise_{name}` to match our current backbones? E.g., `stackwise_se_ratios`, `stackwise_strides` etc

Reply:
Great point; will update this.

Reply:
I went ahead and refactored the names quite a bit across the board.  They should be much more readable now - and make it more obvious what is singular and what is plural.